### PR TITLE
Fix label text alignment in Form Field

### LINKED ISSUE #1810: When in an app that has text-align set to center. FormField's top label inherits this and displays incorrectly
### Package name(s)

Core (@salt-ds/core)

### Package version(s)

1.8.0-rc.0

### Description

When in an app that has text-align set to center. FormField's top label inherits this and displays incorrectly.

This is because text-align isn't explicitly set on the Label.

### Steps to reproduce

https://codesandbox.io/s/salt-ff-input-5hy6c6?file=/src/App.js

### Expected behavior

The top label to be left aligned

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [X] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

### PR DESCRIPTION
**Pull request by libertymayc**
Closes #1810

### CODE REVIEW

File: packages/core/stories/form-field/form-field.stories.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const Skeleton: ComponentStory<typeof FormField> = (props) => {
 
 export const Default: ComponentStory<typeof FormField> = (props) => {
   return (
-    <FlowLayout style={{ width: "366px" }}>
+    <FlowLayout style={{ width: "366px", textAlign: "center" }}>
```
Comment (joshwooding) [inline packages/core/stories/form-field/form-field.stories.tsx:40]: I assume these were only for a test?

```suggestion
    <FlowLayout style={{ width: "366px"  }}>
```

File: packages/core/stories/form-field/form-field.stories.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ export const Label: ComponentStory<typeof FormField> = (props) => {
 
 export const LabelLeft: ComponentStory<typeof FormField> = (props) => {
   return (
-    <FlowLayout style={{ width: "366px" }}>
+    <FlowLayout style={{ width: "366px", textAlign: "center" }}>
```
Comment (joshwooding) [inline packages/core/stories/form-field/form-field.stories.tsx:113]:
```suggestion
    <FlowLayout style={{ width: "366px" }}>
```

File: .changeset/orange-days-guess.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": minor
+---
+
+Fixes label text alignment at component level where value was previously inherited.
```
Comment (joshwooding) [inline .changeset/orange-days-guess.md:5]:
```suggestion
Fixed Form Field label incorrectly inheriting text alignment.
```

**Review by joshwooding — DISMISSED** [review]

File: packages/core/stories/form-field/form-field.stories.tsx (hunk 1 above)
Comment (libertymayc) [reply packages/core/stories/form-field/form-field.stories.tsx:40]: added just to test its global, but yes can remove

**Review by ZarkoPernar — APPROVED** [review]

**Review by ZarkoPernar — APPROVED** [review]

**Pull request merged**